src/modules/controllers/user-controller.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../service/user-service');
+const ApiError = require('../exceptions/api-error');
+const { checkLogin, checkPassword } = require('../../helpers/validation');
+
+class UserController {
+  async registration(req, res, next) {
+    try {
+      const { login, password } = req.body;
+      if (!checkLogin(login) || !checkPassword(password)) {
+        return next(ApiError.BadRequest('validation error'));
+      }
+      const userData = await userService.registration(login, password);
+      res.cookie('refreshToken', userData.refreshToken, { maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true, secure: false });
```

Comment:
where is the accessToken?

Reply:
{ maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true, secure: false }
duplication of code, take it out in constants